dependabot-auto-merge/src/main.ts

```diff
@@ -1,3 +1,7 @@
-console.log( 'hello!' );
+import { mergeDependabotPullRequests } from './autoMerge';
 
-export {};
+mergeDependabotPullRequests()
+	.then()
+	.catch( err => {
+		throw err;
+	} );
```

Comment:
What's the best way to do this? I guess I should exit with `process.exit(1)` but I also want to log the error, no?

Reply:
Planning to `console.error()` it, one for `error.message` and one for `error.stack`